### ISSUE TITLE
replace 2.1 with a new one on upshot.it

### ISSUE THREAD

**Issue by prozacUa** [fwenzel/upshot#74]
Hi there,

Thank you very much for this useful tool!
Can you please update upshot.it to have it pointed on a fresh version of app?

Many thanks!

**Comment by fwenzel** [comment]
Yes! My apologies.

Note that Dropbox is soon going to get rid of the public folder so I will have to introduce changes to upshot to make it work again with other publishing options.

**Comment by prozacUa** [comment]
Yeah, i heard about that. Is Google a good substitution for that?

**Comment by fwenzel** [comment]
You mean Google Drive?

**Comment by fwenzel** [comment]
I am not sure it has an easily usable public URL. Imgur might work well.
